packages/core/src/shader/utils/extractUniformsFromSrc.ts

```diff
@@ -1,19 +1,11 @@
 import { defaultValue } from './defaultValue';
 
-export function extractUniformsFromSrc(vertexSrc, fragmentSrc, mask)
-{
-    const vertUniforms = extractUniformsFromString(vertexSrc, mask);
-    const fragUniforms = extractUniformsFromString(fragmentSrc, mask);
-
-    return Object.assign(vertUniforms, fragUniforms);
-}
-
-function extractUniformsFromString(string)
+function extractUniformsFromString(string: string): any
```

Comment:
The type here is not any.

{[x: string]: {
    type: string;
    dirtyId: number;
    name: string;
    value: number|Float32Array|Int32Array|boolean|boolean[];
};}